db.json

```diff
@@ -6155,6 +6155,9 @@
     "source": "apache",
     "extensions": ["torrent"]
   },
+  "application/x-blender": {
```

Comment:
This file is automated. Changes here will be overridden. The other addition is all that should be necessary.